File: plugins/kotlin/j2k/new/tests/testData/newJ2k/newJavaFeatures/switchExpression/nestedSwitchFallback.java
//method
public void foo(int i, int j) {
  String a =  switch (i) {
        case 0:
            switch (j) {
                case 1:
                    yield "0, 1";
                case 2:
                    yield "0, 2";
            }
        case 1:
            yield "1, x";
        default:    yield "x, x";
}

File: plugins/kotlin/j2k/shared/tests/testData/newJ2k/newJavaFeatures/switchExpression/nestedSwitchFallback.kt
fun foo(i: Int, j: Int) {
    val a = when (i) {
        0 -> {
            when (j) {
                1 -> "0, 1"
                2 -> "0, 2"
            }
            "1, x"
        }

        1 -> "1, x"
        else -> "x, x"
    }
}
